### ISSUE TITLE
[Dungeon] Sethekk Halls bugs

### ISSUE THREAD

**Issue by mhdemn** [Atlantiss/NetherwingBugtracker#2455]
2259

**NORMAL Sethekk Halls**

Sethekk Oracles ([normal](https://tbc-twinhead.twinstar.cz/?npc=18328)) ([heroic](https://tbc-twinhead.twinstar.cz/?npc=20694)) hit real hard with Arcane Lightning ([normal](https://tbc-twinhead.twinstar.cz/?spell=32690)) ([heroic](https://tbc-twinhead.twinstar.cz/?spell=38146)). Will research, can't find anything with quick searches. Consider toning down / weigh in if you find info, please.

![image](https://user-images.githubusercontent.com/25901582/48170871-bec33b80-e2f9-11e8-94fd-ca5cba120603.png)
Not a big difference in damage amongst the group members, seem to hit everybody for nearly the amount shown in the picture.

Time-Lost Controller's ([normal](https://tbc-twinhead.twinstar.cz/?npc=18327)) ([heroic](https://tbc-twinhead.twinstar.cz/?npc=20691)) [totems](https://tbc-twinhead.twinstar.cz/?spell=32764) do nothing. Reckon they mind-controlled a member of the group (not the tank for sure) until totem was destroyed.

- [x] Fixed right away in 2261 - Server crashes everytime the Sethekk Ravenguards ([normal](https://tbc-twinhead.twinstar.cz/?npc=18322)) ([heroic](https://tbc-twinhead.twinstar.cz/?npc=20696)) standing after 1st boss are pulled.

**Comment by Easyfraggy** [comment]
https://github.com/Atlantiss/AtlantissTBCBT/issues/4057

**Comment by mhdemn** [comment]
2304
Wanna update with another detail I've learned now that I was able to finish the instance:

The Shadow Labyrinth Key is currently a drop from Talon King Ikiss.

It should be found inside The Talon King's Coffer, object that spawns in the room when Ikiss dies, and contains a copy of the key for each member of the group; just like The Scarlet Key.

You guys are probably aware of this but I can't find it mentioned in any issue so I'll leave it here just in case.
